libs/image/website/src/lib/image.tsx

```diff
@@ -3,6 +3,7 @@ import {FullImageFragment} from '@wepublish/website/api'
 import {BuilderImageProps} from '@wepublish/website/builder'
 
 declare module 'react' {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Here too

Reply:
this comment is about <T> in react type definition. the T meant for generic type can be removed, but it's recommended to still use it while defining type. so two ways are possible here: so what I did, or just remove it. I prefer the way I did it because I feel it aligns better with typescript's best practices. If you wish, I can just remove it, both approaches are okay

Reply:
No No you are more used to frontend stuff. If you (and the others reviewing it) think it's cleaner, I'm fine with that. :-)

Reply:
<img width="736" alt="image" src="https://github.com/wepublish/wepublish/assets/1131901/42cd91f4-3f76-4bd5-b561-0e2a307187b4">

It's required